python/lsst/meas/algorithms/normalizedCalibrationFlux.py

```diff
@@ -31,6 +32,35 @@
 from .sourceSelector import sourceSelectorRegistry
 
 
+class NormalizedCalibrationFluxError(lsst.pipe.base.AlgorithmError):
+    """Raised if Aperture Correction fails in a non-recoverable way.
+
+    Parameters
+    ----------
+    initNSources : `int`
```

Comment:
Does `init` here mean `initial`? Maybe spell that out explicitly, since these names go into the metadata that is written to disk. `initialNSources` or `nInitialSources` (or `n_initial_sources`, since it seems like we're more fully trying to move to snake_case these days?).

Reply:
I went with `n_initial_sources` and also changed the other variables to snake_case because I realized this file is pretty much all snake case already.